### ISSUE TITLE
New Icon with iOS 11

### ISSUE THREAD

**Issue by albertleao** [AlexDisler/cordova-icon#100]
There is a new marketing icon required with iOS 11:

> Dear developer,
> 
> We have discovered one or more issues with your recent delivery for "xxxxxxxxxxx". Your delivery was successful, but you may wish to correct the following issues in your next delivery:
> 
> Missing Marketing Icon - iOS Apps must include a 1024x1024px Marketing Icon in PNG format. Apps that do not include the Marketing Icon cannot be submitted for App Review or Beta App Review.
> 
> After you’ve corrected the issues, you can use Xcode or Application Loader to upload a new binary to iTunes Connect.
> 
> Regards,
> 
> The App Store team

You can see the requirement here: https://developer.apple.com/ios/human-interface-guidelines/icons-and-images/app-icon/

**Comment by albertleao** [comment]
Pull request here:

#101

**Comment by mcorner** [comment]
Even on 0.10.0 I had to do two things to get past the appstore checks:

1) Remove the alpha channel from the 1024 icon (I have to do this every time i submit).

2) Manually change the Contents.json to include this:

```
    {
      "idiom" : "ios-marketing",
      "size" : "1024x1024",
      "filename" : "icon-1024.png",
      "scale" : "1x"
    },

```

**Comment by shivan** [comment]
Same problem here... is it possible to have cordova-icon remove the alpha on the 1024 icon only? 
The other icons need alpha, dont't they?
Using version 0.11.0

**Comment by WuglyakBolgoink** [comment]
I add small changes to do it. See my PR #109

**Comment by WuglyakBolgoink** [comment]
@mcorner 

> `Manually change the Contents.json to include this:`


cordova-iOS v4.5.3 do it automatically, can you check it please

**Comment by mcorner** [comment]
Yes, I think it assigns it, but it still has an alpha channel until something like #109 gets done, or something better.
